sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -224,22 +225,25 @@ export function readableSpanToEnvelope(
       throw new Error(`Unsupported span kind ${span.kind}`);
   }
 
-  envelope.data.baseData = { ...data, properties, measurements };
+  envelope.data.baseData = { ...data, properties, measurements } as
+    | RequestData
+    | RemoteDependencyData;
   envelope.tags = tags;
-  envelope.time = new Date(hrTimeToMilliseconds(span.startTime)).toISOString();
-  envelope.iKey = instrumentationKey;
-  envelope.ver = 1;
+  envelope.time = new Date(hrTimeToMilliseconds(span.startTime));
+  envelope.instrumentationKey = instrumentationKey;
+  envelope.version = 1;
```

Comment:
These 2 changes will break the exporter until the generated HTTP client is also used with these envelopes (e.g. the client does the serialization from `version` --> `ver`, etc)

Reply:
Looks like the mappers in the new generated code are doing this, maybe we need to couple that change to this one?

Reply:
Done!